frontend/public/components/utils/volume-icon.jsx

```diff
@@ -1,17 +1,19 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
+import { LockIcon, OutlinedCopyIcon, OutlinedFolderOpenIcon } from '@patternfly/react-icons';
+
 import {VolumeSource} from '../../module/k8s/pods';
 
 export const VolumeIcon = ({kind}) => {
-  const faClasses = _.fromPairs([
-    [VolumeSource.emptyDir.id, 'fa-folder-open-o'],
-    [VolumeSource.hostPath.id, 'fa-files-o'],
-    [VolumeSource.secret.id, 'fa-lock'],
+  const icons = _.fromPairs([
+    [VolumeSource.emptyDir.id, <OutlinedFolderOpenIcon className="co-icon-and-text__icon" />],
```

Comment:
This doesn't seem right and is resulting in the following linter error:
```
9:32  error  Missing "key" prop for element in array  react/jsx-key
```

Reply:
This is a false positive from the linter, but `_.fromPairs` isn't needed here. I'd suggest....

```tsx
const icons = {
  [VolumeSource.emptyDir.id]: <OutlinedFolderOpenIcon className="co-icon-and-text__icon" />,
  // ...
};
```

Reply:
Thanks, @spadgett.  Updated.